src/platform-implementation-js/dom-driver/gmail/widgets/gmail-mole-view-driver.ts

```diff
@@ -104,7 +104,24 @@ class GmailMoleViewDriver {
 
   show() {
     const doShow = (moleParent: HTMLElement) => {
-      moleParent.insertBefore(this.#element, last(moleParent.children)!);
+      const lastChild = last(moleParent.children)!;
+
+      if (lastChild instanceof HTMLElement) {
+        /**
+         * 2023-10-02 When Google Chat is enabled, we need to keep track of the `order` style property for moles added.
+         * If we don't, we end up with moles added on top of the sidebar because the moleParent is using `order` for layout, and any child without `order` set will be forced to the right edge of the page.
+         *
+         * This fix ratches the order of the element padding in order to present moles before the sidebar. Whether or not this fix works with Gmail's CSS in a RTL language is unknown.
```

Comment:
```suggestion
         * This fix ratchets the order of the element padding in order to present moles to the left the right sidebar.
```

I checked against Hebrew language Gmail (which I can't read) that `order` and the SDK's mole insertion into the DOM still work. It appears that they do.